pkg/helm/charts/tests/centralservices/testdata/helmtest/central.test.yaml

```
@@ -89,12 +71,10 @@ tests:
   values:
     env:
       openshift: 4
-    system:
-      createSCCs: false
   expect: |
     .roles["use-central-scc"] | assertThat(. != null)
     .rolebindings["central-use-scc"] | assertThat(. != null)
-    .securitycontextconstraints["stackrox-central"] | assertThat(. == null)
+    .securitycontextconstraints["restricted-v2"] | assertThat(. == null)
```

Central is now on `nonroot-v2`
```suggestion
    .securitycontextconstraints["nonroot-v2"] | assertThat(. == null)
```